packages/forms/stories/static/stringCompletionItems.json

```diff
@@ -0,0 +1,18 @@
+[{
+	"title": "A",
```

Comment:
I'm wondering, this JSON structure is needed by component but I think we can mask complexity for widget?

```json
[
  { "group": "A", "items": [ "Apple", "Apricot"] },
  { "group": "B", "items": [ "Banana"] },
  { "group": "C", "items": [ "Cherry"] }, 
]
```